tests/test_EclipseIO.cpp

```diff
@@ -314,9 +317,19 @@ BOOST_AUTO_TEST_CASE(EclipseIOIntegration) {
             { "TRANZ", { measure::transmissibility, tranz, TargetType::INIT } },
         };
 
+        std::map<std::string, std::vector<int> > map;
```

Comment:
This can alternatively be initialized as:
```C++
std::map<std::string, std::vector<int>> int_data =  {{"STR_ULONGNAME" , {1,1,1,1,1,1,1,1,1,1,1,1,......}}};
```